content/blog/2023/when-to-choose-a-responsive-web-app.md

```diff
@@ -0,0 +1,160 @@
+
+Search engines, like Google, crawl the web looking for keywords and other information to sort and rank the content. Then they collect, process, and store a database of all that content. This is called *indexing*. RWAs live on the web, so unless special code is added to prevent it, they will automatically be indexed and appear in organic search results.
+
+### Broad Reach
+*…when you want to reach a broad audience across devices.*
+
+People can access the same RWA on any device from a small iPhone to a tablet, from a laptop to a giant Windows desktop, unlike native mobile apps which are limited to supported mobile devices. Additionally, the web provides a high level of backward compatibility, meaning RWAs can be available in some form even on old devices. On the other hand, one platform update can totally block a device from accessing a native mobile app.
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2023/no.png',
+    alt: 'The word no with a frowning face'
+  }],
+  attrs={
+    'style': '--column: content-start'
+  }
+) }}
+
+## Don’t choose an RWA when…
+### Limited Capabilities
+*…when your project requires a high level of specialized capability.*
+
+RWAs cannot access all of a device’s hardware and software features. For example, if your digital product requires direct integration with a specific mobile device’s GPS for geofencing or access to light and proximity sensors, then an RWA won’t work.
+
+It is *possible* to access some advanced device features in the context of a RWA, but then you would be moving into the [progressive web app](/2023/07/05/when-to-choose-a-progressive-web-app/) space, not strictly an RWA which provides static content for people to consume.
+
+
+
+### Performance Limitations
+*…when you need advanced performance.*
+
+If a person has a slow internet connection, they will have trouble accessing RWA content that requires a lot of processing power – like videos or animated games. If they have no internet connection, they will not be able to access an RWA at all.
+
+### Security Limitations
+*…when you require advanced security.*
+
+If your project needs advanced security beyond typical HTTPS encryption, such as with banking applications, an RWA is less likely able to meet those requirements.
+
+### Not Discoverable Via App Store
+*…when the people you want to reach will look for your app in an app store.*
+
+RWAs are not listed in app stores. If the people you want to reach exclusively use app stores to search for apps, they will not be able to find a responsive web app.
+
+
+## In Summary
+A responsive web app tends to be less expensive to build than other types of apps -- especially the native mobile app -- and offers the broadest reach. But if have specialized needs around performance, security, or the app stores, one of the other two types of apps -- a native mobile app or a progressive web app -- are probably going to be a better fit. Let's take a look at the pros and cons of the native mobile app next.
```

Comment:
> Let's take a look…

That active phrasing makes it sound like we're going right into it, but the reality is we're waiting a month.

Reply:
The last line of both page 1 and 2 in the summary can probably be commented out until those posts are live IMO